pyproject.toml

```diff
@@ -48,3 +48,11 @@ ignore = [
 [tool.ruff.per-file-ignores]
 "gcp/shap-e/app.py" = ["E402"] # module level import not at top of file -- expected for sys.path.insert() usage
 
+[tool.ruff.lint.isort]
+known-third-party = ["modal"]
+known-first-party = [
+    "punctuator",
+    "runner",
+    "shared",
+    "tuner",
+]
```

Comment:
this allows for consistently splitting the imports into std/dependency/first-party groups

given the slightly non-standard project structure (and the re-use of `modal` as a directory name & also a dependency name)  i think it was isorting implicitly & was different on my local vs in CI

Reply:
I wonder if there's a better way to structure our project? Can ruff target or isort has sub projects?